ui/basicTab/basicTab.pug

```diff
@@ -0,0 +1,58 @@
+include ../global/advantages.pug
+
+// Basic section 
+.basic.section
+  = '\n'
+  | Race 
+  input.wide(name='attr_race' type='text')
+  = '\n'
+  |   Gender 
+  input.wideish(name='attr_gender' type='text')
+  = '\n'
+  |   Height 
+  input.wideish(name='attr_height' type='text')
+  = '\n'
+  |   Weight 
+  input.wideish(name='attr_weight' type='number')
+  = '\n'
+  |   Age 
+  input.wideish(name='attr_age' type='number')
+  .horizontal-flex
+    // Advantages and Disadvantages 
+    .advantages.flex-stack
+      input(name='attr_advantage_total_CP' type='hidden' value='0')
+      .advantage-row.table-header
+        div Advantage
+        .count-placeholder
+        .CP CP
+      each advantage in advantages
+        .advantage-row
+          div= advantage.description
+          if advantage.minusAdvantageFieldName
+            input(name=advantage.minusAdvantageFieldName type='checkbox' value='1' hidden='true')
+          select.skillexpertise(name=advantage.fieldName)
+            option(value=' ')
+            each optionValue in advantage.skillExpertiseOptions || [-1, 1, 2, 3]
+              option(value=optionValue)=optionValue
+          span.CP(name=advantage.cpFieldName)
+      fieldset.repeating_extraadvantage
+        //
+          - We leave no space between the inputs, so there won't be
+          any inside the repeating section
```

Comment:
This comment isn't right any more, because pug is putting in spaces. So it should go away.
I had originally crammed everything on one line because it was the only way I figured out to keep extra space from appearing between the inputs. With grid, I assume we can do better.
(I'm one of those people who play on a single laptop, so I hate seeing pixels that aren't giving me any information.)

Reply:
I'll remove this comment.
Yeah, I think grid will help, and we can also adjust gaps and such.